### DISCUSSION TITLE
Removing download link

### DISCUSSION THREAD

**Discussion by Rasmus10** [Lumieducation/H5P-Nodejs-library#1555] [Q&A]
After creating h5p component it looks like this.
![image](https://user-images.githubusercontent.com/57188630/124590498-e03aa380-de63-11eb-8c28-158ded667ff9.png)

Is it possible to remove this download link form here?

**Comment by sr258** [comment]
You have to replace the default server-side renderer with your own by calling H5PPlayer.setRenderer. You can check out the default renderer here: https://github.com/Lumieducation/H5P-Nodejs-library/blob/74dd1d6d3ab59cb36dad21728363cb68e0f8804f/packages/h5p-server/src/renderers/player.ts
